edx/edx-enterprise
# ENT-47: Link the EnterpriseCustomer model to the Site model

Hi @mattdrayer , @e-kolpakov , @asadiqbal08 

Kindly take a look at this PR, This PR adds an association between `Site` and `EnterpriseCustomer` model.

__Testing instructions:__

1.  This PR adds only a `ForeignKey` relationship to `EnterpriseCustomer` with updates to django admin interface - smoke testing Django admin using Enterprise Customer should be enough.
    - Open "Enterprise Customer" page in Django admin
    - Try adding a new `Enterprise Customer`, try adding an "Enterprise Customer" with and without selecting a site. ("Enterprise Customer" addition without a site will error out)

2. Django admin features to export list of Enterprise Customers to CSV is updated and CSV should contain site info.
    - Open "Enterprise Customer" page in Django admin
    - Select all/some Enterprise Customers.
    - In "Actions" dropdown select "Export to CSV", than press "Go" button. Expected result: selected Enterprise Customers are exported to CSV
    - Make sure that exported CSV now has "site" column with site domain as value.
